end-to-end-tests/test-app/server.js

```diff
@@ -1,7 +1,9 @@
 const express = require('express');
 const path = require('path');
 const app = express();
-const port = 3000;
+
+const PORT = process.env.PORT || 3000;
+const SERVICE_WORKER_MISSING = process.env.SERVICE_WORKER_MISSING || 'false';
```

Comment:
Feels weird to have a negative env var, `SERVICE_WORKER_PRESENT === 'false'` makes more sense than `SERVICE_WORKER_MISSING === 'true'`